### PR TITLE
move reload and stop buttons, change homepage

### PR DESCRIPTION
**Pull request by PalmerAL**
-move the reload and stop buttons so that they are in the input bar
screen shot 2015-01-05 at 12 42 51 pm

### CODE REVIEW

File: apps/browser/css/navbar.css [hunk 1]
```diff
@@ -38,7 +38,14 @@
   border-radius: 3px;
 }
 
-.navbar > button.disabled {
+.urlbar > button {
+ background-image: url(images/shared/reload-stop-go.png);
+ background-size: 42px 28px;
+ width:12px;
```
Comment (paulrouget) [inline apps/browser/css/navbar.css]: missing space.

File: apps/browser/css/navbar.css [hunk 2]
```diff
@@ -38,7 +38,14 @@
   border-radius: 3px;
 }
 
-.navbar > button.disabled {
+.urlbar > button {
+ background-image: url(images/shared/reload-stop-go.png);
```
Comment (paulrouget) [inline apps/browser/css/navbar.css]: Does this look good with the blacktheme?

File: apps/browser/css/navbar.css [hunk 3]
```diff
@@ -38,7 +38,14 @@
...
+ background-size: 42px 28px;
+ width:12px;
+ height: 12px;
```
Comment (paulrouget) [inline apps/browser/css/navbar.css]: Isn't the size of the icons 14px?

File: apps/browser/css/navbar.css [hunk 4]
```diff
@@ -78,6 +88,14 @@
...
 .stop-button {    background-position: -90px   0px; }
 .menu-button {    background-position: -468px  0px; }
+  
+   /* styles for the urlbar buttons */
+
+.urlbar > .reload-button { background-position: 2px -1px; }
+.urlbar > .reload-button:active { background-position: 2px -15px; }
+.urlbar > .stop-button { background-position: -12px 0px; }
+.urlbar > .stop-button:active { background-position: -12px -14px; }
+
```
Comment (paulrouget) [inline apps/browser/css/navbar.css]: These are weird positions. Use the proper coordinates of the images (0px/14px 0px/14px).

File: apps/browser/css/navbar.css (hunk 2 above)
Comment (PalmerAL) [inline apps/browser/css/navbar.css]: pretty good. the image is a medium-gray, so it stands out on either a light or dark background.
screen shot 2015-01-07 at 3 44 43 pm

File: apps/browser/css/navbar.css (hunk 3 above)
Comment (PalmerAL) [inline apps/browser/css/navbar.css]: They are in fact 12x12px. (The regular icons from toolbar.png are too large to fit in the urlbar, so these 12x12px ones are used instead)

**Comment by paulrouget** [comment]
Thanks!

**Comment by jdm** [comment]
@PalmerAL You probably want to change your git settings to add some sort of author information besides "Your Name <your@email.com>"

**Pull request merged**